src/test/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSourceUnitTest.java

```diff
@@ -85,19 +85,6 @@ public void testHeadersAreStripped() {
         }
     }
 
-    @Test
-    public void testPartitionSizing(){
-
-        String bam = dirBQSR + "HiSeq.1mb.1RG.2k_lines.alternate.bam"; //file is ~220 kB
-        JavaSparkContext ctx = SparkContextFactory.getTestSparkContext();
-
-        ReadsSparkSource readSource = new ReadsSparkSource(ctx);
-        JavaRDD<GATKRead> allInOnePartition = readSource.getParallelReads(bam, null);
-        JavaRDD<GATKRead> smallPartitions = readSource.getParallelReads(bam, null, ReadsSparkSource.DEFAULT_SPLIT_SIZE / 100);
-        Assert.assertEquals(allInOnePartition.partitions().size(), 1);
-        Assert.assertEquals(smallPartitions.partitions().size(), 2);
-    }
-
```

Comment:
Of course we will ask for some new unit tests once this branch settles on a final design :)